packages/apollo-fragment-react/example/package.json

```diff
@@ -7,23 +7,18 @@
     "react-scripts": "1.0.16"
   },
   "dependencies": {
-    "apollo-cache-inmemory": "^1.0.0",
-    "apollo-client": "^2.3.1",
+    "@apollo/client": "^3.5.6",
     "apollo-fragment-react": "^0.1.0",
-    "apollo-link": "^1.0.0",
     "apollo-link-state": "^0.4.1",
     "apollo-link-state-fragment": "^0.1.0",
```

Comment:
Jury is still out on what to do with these deps. Will circle back after having a conversation